src/syntax/FStar.Syntax.Resugar.fs

```diff
@@ -1095,16 +1095,19 @@ and resugar_arg_qual env (q:option<S.arg_qualifier>) : option<(option<A.arg_qual
     if b then None
     else Some (Some A.Implicit)
   | Some S.Equality -> Some (Some A.Equality)
-  | Some (S.Meta t) ->
-    Some (Some (A.Meta (resugar_term' env t)))
+  | Some (S.Meta (S.Arg_qualifier_meta_tac t)) ->
+    Some (Some (A.Meta (A.Arg_qualifier_meta_tac (resugar_term' env t))))
+  | Some (S.Meta (S.Arg_qualifier_meta_attr t)) ->
+    Some (Some (A.Meta (A.Arg_qualifier_meta_attr (resugar_term' env t))))
 
 and resugar_imp env (q:option<S.arg_qualifier>) : A.imp =
   match q with
   | None -> A.Nothing
   | Some (S.Implicit false) -> A.Hash
   | Some S.Equality
   | Some (S.Implicit true) -> A.Nothing // We don't have syntax for inaccessible arguments
-  | Some (S.Meta t) -> A.HashBrace (resugar_term' env t)
+  | Some (S.Meta (S.Arg_qualifier_meta_tac t)) -> A.HashBrace (resugar_term' env t)
+  | Some (S.Meta (S.Arg_qualifier_meta_attr t)) -> A.Nothing //TODO: Review this 03/31
```

Comment:
I think it's indeed fine to return `A.Nothing`, since these implicits come only from applications. Then the case above also seems useless.. I think I added it, I could remove it later.